src/app/forms/custom-sections/adr-tank-statement-un-number-edit/adr-tank-statement-un-number-edit.component.spec.ts

```diff
@@ -112,8 +82,11 @@ describe('AdrTankStatementUnNumberEditComponent', () => {
     });
 
     it('should prevent the adding of additional controls, when the previous one is empty', () => {
+      component.formArray = fb.array<CustomFormControl>([]);
+      component.addControl('valid');
       component.addControl();
-      expect(component.formArray.value).toHaveLength(1);
+      component.addControl();
+      expect(component.formArray.value).toHaveLength(2);
     });
```

Comment:
i think this test description needs updating to encapsulate the behaviour of this test now as its not checking to see if adding an extra control is prevented, but rather expecting the opposite and that it has been added